src/Common/IntervalKind.h

```diff
@@ -33,10 +33,17 @@ struct IntervalKind
     /// For `Month`, `Quarter` and `Year` the function returns an average number of seconds.
     Int32 toAvgSeconds() const;
 
+    /// Returns exact number of seconds in one interval.
```

Comment:
(nit: could have listed the "base case" (toSeconds) before the "advanced case" (toAvgSeconds))